osu.Game/Overlays/SettingsToolboxGroup.cs

```diff
@@ -156,6 +156,8 @@ protected override bool OnInvalidate(Invalidation invalidation, InvalidationSour
 
         private void updateExpandedState(ValueChangedEvent<bool> expanded)
         {
+            content.ClearTransforms();
```

Comment:
Probably wouldn't leave this here without an inline comment about what this is trying to do.

Reply:
It may be obvious to you and me now, but it won't be in N months. Also I don't know about handling it in auto size setter, I think it's better to fail loudly rather than try and compensate the user intent *might* have been.

Reply:
I've pushed one, feel free to modify it to your liking.